src/index.js

```diff
@@ -38,4 +39,8 @@ angular.module('vlui', [
       T: 'time',
       G: 'geo'
     }
+  })
+  .config(function (AnalyticsProvider) {
+    AnalyticsProvider
+      .setAccount({ tracker: 'UA-44428446-4', name: 'voyager', trackEvent: true });
```

Comment:
My only concern here is that this will make all consumers of `vega-lite-ui` use this tracking ID; it's not hugely likely for now that third parties would be using these tools, but since there's no formal reason why they would not be able to, this feels like it has the potential to mess up our stats.